app/models/article.rb

```diff
@@ -9,12 +9,14 @@
 #  updated_at :datetime         not null
 #
 class Article < ApplicationRecord
-    has_many :comments,
+    has_many(
+    :comments,
     dependent: :destroy
-    validates :title,
-    presence: true,
-    length: { minimum: 5 }
-    validates :text,
-    presence: true,
-    length: { minimum: 5 }
+    )
```

Comment:
incorrect alignement,
all parameters going within a block, or a `(` `)` need to be intended by with 1 block.
```
has_many(
  :comments,
  dependent: :destroy
)
```